### PR TITLE
AQCU-1580: Add "period" to derivation chain popup info per node.

### PR DESCRIPTION
**Pull request by kmschoep-usgs**

### CODE REVIEW

File: man/makeGapsInclusiveIfStatistic.Rd [hunk 1]
```diff
@@ -0,0 +1,20 @@
+% Generated by roxygen2: do not edit by hand
```
Comment (mhines-usgs) [inline man/makeGapsInclusiveIfStatistic.Rd:1]: lol, i just keep ignoring this file in my Rstudio.. good for you.

Comment (kmschoep-usgs) [reply man/makeGapsInclusiveIfStatistic.Rd:1]: I never know what to include in my commits for R, so went for it...

**Review by mhines-usgs — APPROVED** [review]
:+1: looks good to me

**Comment by mhines-usgs** [comment]
This will actually be a good test to see if this makes its way out to reporting-test!

**Pull request merged**